Python/Tests/Analysis/ParserTests.cs

```diff
@@ -2653,6 +2669,40 @@ public class ParserTests {
             }
         }
 
+        [TestMethod, Priority(0)]
```

Comment:
Currently we don't parse the content at all - we simply ignore the prefix and treat it like a regular string. (This is actually how Python handles it as well, which is suboptimal, but not trivial to solve right now...)